### DISCUSSION TITLE
SoundSwitch Crashing after PC is Sleeping

### DISCUSSION THREAD

**Discussion by Blake-V-Walker** [Belphemur/SoundSwitch#796] [Q&A] [answered]
Hi All - Whenever I come back to my PC after having it slept for a while, my SoundSwitch has crashed without fail.

Is anyone else having this issue? 

Clicking "Yes" to share the error with the developers just takes me to the main Github page. 

I have reinstalled with the same issues. 

![image](https://user-images.githubusercontent.com/73499422/137659628-6fe47986-ae52-457c-bedb-fbcb6220c8bd.png)

**Comment by Belphemur** [comment, marked as answer]
I moved this into a proper issue:
https://github.com/Belphemur/SoundSwitch/issues/797
